exercises/hamming/hamming_test.exs

```diff
@@ -8,34 +8,46 @@ ExUnit.configure exclude: :pending, trace: true
 defmodule DNATest do
   use ExUnit.Case
 
-  # @tag :pending
+#   @tag :pending
+  test "hamming distance for strands of same length returns :ok" do
+    {code, _} = DNA.hamming_distance('', '')
+    assert code == :ok
+  end
+
+  @tag :pending
+  test "hamming distance is undefined for strands of different lengths" do
+    assert DNA.hamming_distance('AAAC', 'TAGGGGAGGCTAGCGGTAGGAC') == {:error, "Lists must be the same length."}
+    assert DNA.hamming_distance('GACTACGGACAGGACACC', 'GACATCGC') == {:error, "Lists must be the same length."}
+  end
+
+  @tag :pending
   test "no difference between empty strands" do
-    assert DNA.hamming_distance('', '') == 0
+    {:ok, distance} = DNA.hamming_distance('', '')
+    assert distance == 0
```

Comment:
my first inclination was to pull out a helper assertion, I just wasn't sure that exercism community wanted the complication in an effort to keep the tests as "Plain Jane" as possible. I'll take a stab with the suggestions and see what you guys think! I'll probably keep the expected / actual on the same `lhs` `rhs` of the assertion, since that is what users are used to through the previous exercises.